index.html

```diff
@@ -40,6 +41,170 @@ <h2>I'am a Software Developer</h2>
+                    </ul>
+                    <button class="project-button mobile-button">See Project</button>
+                </div>
+            </article>
+            <article class="project-card mobile-card">
+                <div class="transparent">
+                    <div class="project-desc mobile-desc">
+                        <h3 class="desc-title">Multi-Post Stories</h3>
+                        <p class="description mobile-description">
+                            A daily selection of privately personalized reads;
+                            no accounts or sign-ups required. has been 
+                            the industry's standard dummy text ever since the 1500s, 
+                            when an unknown printer took a standard dummy text.
+                            </p>
+                    </div>
+                    <ul class="project-tech mobile-tech">
+                        <li><a href="#">html</a></li>
+                        <li><a href="#">bootstrap</a></li>
+                        <li><a href="#">Ruby</a></li>
+                    </ul>
+                    <button class="project-button mobile-button">See Project</button>
+                </div>
+            </article>
+            <article class="project-card mobile-card">
+                <div class="transparent">
+                    <div class="project-desc mobile-desc">
+                        <h3 class="desc-title">Multi-Post Stories</h3>
+                        <p class="description mobile-description">
+                            A daily selection of privately personalized reads;
+                            no accounts or sign-ups required. has been 
+                            the industry's standard dummy text ever since the 1500s, 
+                            when an unknown printer took a standard dummy text.
+                        </p>
+                    </div>
+                    <ul class="project-tech mobile-tech">
+                        <li><a href="#">html</a></li>
+                        <li><a href="#">bootstrap</a></li>
+                        <li><a href="#">Ruby</a></li>
+                    </ul>
+                    <button class="project-button mobile-button">See Project</button>
+                </div>
+            </article>
+        </section>
+        <section class="about-section">
+            <header class="about-header">
+                <h3>About Me</h3>
+            </header>
+            <article class="about-desc">
+                <p class="description mobile-description">
+                    A daily selection of privately personalized reads;
+                    no accounts or sign-ups required. has been 
+                    the industry's standard dummy text ever since the 1500s, 
+                    when an unknown printer took a standard dummy text.
+                </p>
+                <div class="abt-bottom">
+                    <div class="project-button about-btn">My Resume</div>
```

Comment:
- [Optional] Please this should be a link of `<a>` or a  `<button>` tag, as it will allow the user to be able to download your Resume.

- [Optional] Please work on the padding that surrounds the `My Resume`
- [OPtional] Please work on your margin in your about section to be similar to that of the Figma design
- [Optional] Please make the text be `Get My Resume` as it is on the Figma design

##### Your design
![Screenshot from 2022-05-27 19-32-01](https://user-images.githubusercontent.com/58771507/170771043-4faf534e-03fe-4758-bc3b-3a17b56b6c44.png)

##### Figma design
![Screenshot from 2022-05-27 19-32-28](https://user-images.githubusercontent.com/58771507/170771102-9d2bf002-af13-4722-8866-d52f955568fb.png)